config/example/config.txt

```diff
@@ -439,6 +439,8 @@ HIGH_POP_MC_MODE_AMOUNT 65
 ##Disengage high pop mode if player count drops below this
 DISABLE_HIGH_POP_MC_MODE_AMOUNT 60
 
+##Developer options
```

Comment:
maybe this should have a bunch of hyphens to make it stand out more like ------------------------ Developer Options -------------

Reply:
Probably. Not sure what's the best way to make it stand out and standardize it in the file though.